nevergrad/functions/irrigation/known_geoloc.json

```diff
@@ -0,0 +1,46 @@
+{
```

Comment:
this is actually better than hard coding it in code ✨

Reply:
aha thanks! I think this code is quite unreadable and I will do some additional refactoring to make it more understandable.